app/frontend/javascript/custom_google_analytics_4_events.js

```diff
@@ -6,7 +6,7 @@
 $(document).on('click', '*[data-analytics-category]', function(e) {
 
```

Comment:
The comment below is wrong. As you might imagine,
JS function `gtag` is not defined in
`app/views/layouts/_google_analytics_universal.html.erb .`
It's defined in 
`app/views/layouts/_google_analytics_4.html.erb`.